core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/containerModels.scala

```diff
@@ -1,51 +1,47 @@
 package org.broadinstitute.dsde.workbench.leonardo
 
 import ca.mrvisser.sealerate
+import enumeratum.{Enum, EnumEntry}
 
 import scala.util.matching.Regex
 
 /** Container registry, e.g. GCR, Dockerhub */
-sealed trait ContainerRegistry extends Product with Serializable {
+sealed trait ContainerRegistry extends EnumEntry with Product with Serializable {
```

Comment:
this data type already has `allRegistries`, why are we adding `EnumEntry`?

Reply:
I added it so that I could use `withNameInsensitiveOption` for the Decoder and `entryName` for the Encoder. Is there a drawback to using `EnumEntry`?

Reply:
it's just duplicate since we're using sealed trait there as well...for `withNameInsensitiveOption`, we can use all lowercase in the string representation, and in decoder, just do `_.toLowerCase`